retroshare-gui/src/gui/statistics/GxsTransportStatistics.cpp

```diff
@@ -146,6 +152,19 @@ void GxsTransportStatistics::CustomPopupMenu( QPoint )
 	contextMnu.exec(QCursor::pos());
 }
 
+void GxsTransportStatistics::CustomPopupMenuGroups( QPoint )
+{
+	QMenu contextMnu( this );
+	
+	QTreeWidgetItem *item = groupTreeWidget->currentItem();
+	if (item) {
+	contextMnu.addAction(QIcon(":/images/info16.png"), tr("View details"), this, SLOT(showAuthorInPeople()));
```

Comment:
Please use FileDefs to create the icon! It should be the default now for all icons created from files.

Reply:
Also if you see some explicit icon from file lines, dont hesitate to convert them.